crates/rpc-proxy/src/bin/main.rs

```diff
@@ -55,6 +61,11 @@ enum Commands {
 #[derive(Parser, Debug)]
 struct ServerArgs {
     // ========== General Configuration ==========
+    /// Address to bind to
+    /// Example: --bind 0.0.0.0
+    #[arg(long, default_value = "127.0.0.1")]
+    bind_address: String,
+
```

Comment:
I think `host` is a more idiomatic name.
```suggestion
    /// Address to bind to
    /// Example: --name 0.0.0.0
    #[arg(long, default_value = "127.0.0.1")]
    host: String,

```

Reply:
updated!